tests/api/v1/generator/base_case.py

```diff
@@ -91,6 +91,7 @@ def _template_data(self):
         return {
             'user': os.environ.get('USER'),
             'workingDirectory': os.environ['PTERO_WORKFLOW_TEST_TEMP'],
+            'environment': json.dumps(dict(os.environ)),
```

Comment:
The reason we had to dump the environment into the template was to support executing the wrapper script, so that the wrapper script would use the right python and find its dependencies (such as the requests module).  If we can get rid of the wrapper script, we can avoid this environment stuff for now.